website/server/models/group.js

```diff
@@ -596,7 +596,16 @@ schema.methods._processCollectionQuest = async function processCollectionQuest (
     return m;
   }, []);
 
-  foundText = foundText.length > 0 ? foundText.join(', ') : 'nothing';
+  // Text for no found quest items
+  let questItems = Object.keys(this.quest.progress.collect).map((item) => {
+    return quest.collect[item].text('en');
+  });
+  questItems.sort(); // Make sure message is always reported in the same order
+
+  let noFoundQuestItems = '0 ';
+  noFoundQuestItems += questItems.join(', 0 ');
```

Comment:
This logic does not work for translations because it assumes a similar word order to English.  Also, it looks like the questItems are all in English (i.e. not translated) anyway?